File: src/main/java/uk/co/lukestevens/encryption/IgnoredEncryptionService.java
package uk.co.lukestevens.encryption;

import java.io.IOException;


public class IgnoredEncryptionService implements EncryptionService {

	@Override
	public String encrypt(String plaintext) throws IOException {
		return plaintext;
	}

	@Override
	public String decrypt(String ciphertext) throws IOException {
		return ciphertext;
	}

}

File: src/main/kotlin/uk/co/lukestevens/encryption/IgnoredEncryptionService.kt
package uk.co.lukestevens.encryption

import java.io.IOException

class IgnoredEncryptionService : EncryptionService {
	
	override fun encrypt(plaintext: String): String {
		return plaintext
	}

	override fun decrypt(ciphertext: String): String {
		return ciphertext
	}
}